src/smalloc.cc

```diff
@@ -152,19 +166,9 @@ size_t ExternalArraySize(enum ExternalArrayType type) {
 void CopyOnto(const FunctionCallbackInfo<Value>& args) {
   Environment* env = Environment::GetCurrent(args);
 
-  if (!args[0]->IsObject())
-    return env->ThrowTypeError("source must be an object");
-  if (!args[2]->IsObject())
```

Comment:
Mind switching these to `ASSERT()`? Useful for when we're calling these directly from core.